tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -26,9 +26,26 @@
 class OperationCtx:
     def __init__(self, redpanda):
         self.redpanda = redpanda
+        self.brokers_list = redpanda.brokers_list()
+        self.cur_first_broker = itertools.cycle(self.brokers_list)
+
+    def _get_brokers(self):
+        # This is a bit of an ugly hack to deal with the consequences of
+        # https://github.com/redpanda-data/redpanda/issues/6317
+        # If the first broker in the list is suspended, rpk won't try other
+        # brokers and will fail straight away. This is a problem even if we
+        # shuffle the brokers list (as redpanda.brokers_list() does) and add
+        # retries - there is a small chance that the first broker will be the
+        # bad one in all retries! To mitigate this we deteriministically choose
+        # the first broker so that it is always different from the previous
+        # invocation and shuffle the rest.
```

Comment:
thanks for writing this comment.

Reply:
> This is a problem even if we shuffle the brokers list (as redpanda.brokers_list() does) and add retries - there is a small chance that the first broker will be the bad one in all retries!

curious about this. it makes sense, but it is also surprising. do we not have enough retries and/or the timeouts are too long to support enough retries to make the chances tiny enough?

Reply:
The default in `admin_ops_fuzzer` is 5 retries, but some operations execute rpk twice (and fail if either one of the commands fail), so the test fails from time to time. I guess adding 10 retries would have worked as well? A bit hard for me to calculate the probabilities :)